keyboards/jukaie/jk01/info.json

```diff
@@ -0,0 +1,225 @@
+            { "flags": 4, "matrix": [6, 1], "x": 125, "y": 20 },
+            { "flags": 4, "matrix": [10, 2], "x": 135, "y": 20 },
+            { "flags": 4, "matrix": [6, 5], "x": 150, "y": 20 },
+            { "flags": 4, "matrix": [2, 1], "x": 0, "y": 30 },
+            { "flags": 4, "matrix": [1, 2], "x": 17, "y": 30 },
+            { "flags": 8, "matrix": [2, 2], "x": 27, "y": 30 },
+            { "flags": 4, "matrix": [3, 2], "x": 37, "y": 30 },
+            { "flags": 4, "matrix": [4, 2], "x": 47, "y": 30 },
+            { "flags": 4, "matrix": [4, 3], "x": 57, "y": 30 },
+            { "flags": 4, "matrix": [5, 3], "x": 67, "y": 30 },
+            { "flags": 4, "matrix": [5, 2], "x": 77, "y": 30 },
+            { "flags": 4, "matrix": [6, 2], "x": 87, "y": 30 },
+            { "flags": 4, "matrix": [7, 2], "x": 97, "y": 30 },
+            { "flags": 4, "matrix": [8, 2], "x": 107, "y": 30 },
+            { "flags": 4, "matrix": [8, 3], "x": 117, "y": 30 },
+            { "flags": 4, "matrix": [10, 4], "x": 127, "y": 30 },
+            { "flags": 4, "matrix": [1, 5], "x": 150, "y": 30 },
+            { "flags": 4, "matrix": [0, 0], "x": 0, "y": 40 },
+            { "flags": 4, "matrix": [1, 4], "x": 22, "y": 40 },
+            { "flags": 4, "matrix": [2, 4], "x": 32, "y": 40 },
+            { "flags": 4, "matrix": [3, 4], "x": 42, "y": 40 },
+            { "flags": 4, "matrix": [4, 4], "x": 52, "y": 40 },
+            { "flags": 4, "matrix": [4, 5], "x": 62, "y": 40 },
+            { "flags": 4, "matrix": [5, 5], "x": 72, "y": 40 },
+            { "flags": 4, "matrix": [5, 4], "x": 82, "y": 40 },
+            { "flags": 4, "matrix": [6, 4], "x": 92, "y": 40 },
+            { "flags": 4, "matrix": [7, 4], "x": 102, "y": 40 },
+            { "flags": 4, "matrix": [8, 5], "x": 112, "y": 40 },
+            { "flags": 4, "matrix": [9, 1], "x": 122, "y": 40 },
+            { "flags": 4, "matrix": [3, 5], "x": 140, "y": 40 },
+            { "flags": 4, "matrix": [2, 5], "x": 150, "y": 40 },
+            { "flags": 4, "matrix": [0, 6], "x": 0, "y": 50 },
+            { "flags": 4, "matrix": [9, 0], "x": 12, "y": 50 },
+            { "flags": 4, "matrix": [9, 3], "x": 25, "y": 50 },
+            { "flags": 4, "matrix": [9, 4], "x": 37, "y": 50 },
+            { "flags": 4, "matrix": [9, 5], "x": 100, "y": 50 },
+            { "flags": 4, "matrix": [9, 2], "x": 110, "y": 50 },
+            { "flags": 4, "matrix": [0, 4], "x": 120, "y": 50 },
+            { "flags": 4, "matrix": [0, 3], "x": 130, "y": 50 },
+            { "flags": 4, "matrix": [7, 3], "x": 140, "y": 50 },
+            { "flags": 4, "matrix": [0, 5], "x": 150, "y": 50 },
+            { "flags": 4, "matrix": [11, 0], "x": 0, "y": 1 },
+            { "flags": 4, "matrix": [11, 1], "x": 0, "y": 2 }
+        ]
+    },
+    "url": "",
+    "usb": {
+        "device_version": "0.0.2",
+        "pid": "0x0002",
+        "vid": "0x0191"
```

Comment:
While this VID wasn't in use, it's very low in the assigned ranges, and in a spot that is unused.  That makes me uncomfortable.  

Ideally, the higher this value is, the better.  And the further above 0x4000, the better.